safe-global/safe-wallet-web
# [ SW-501 ] fix: Update privacy policy

## What it solves

Updating the privacy policy according to [this request](https://5afe.slack.com/archives/C045P36MXQC/p1730108275091469)

## How this PR fixes it

- Adds markdown support for privacy policy

## How to test it

1. Open the privacy page
2. Observe the updates from the document

## Checklist
